src/Shared/runtime/Http2/Hpack/HeaderField.cs

```diff
@@ -36,7 +36,7 @@ public override string ToString()
         {
             if (Name != null)
             {
-                return Encoding.ASCII.GetString(Name) + ": " + Encoding.ASCII.GetString(Value);
+                return Encoding.Latin1.GetString(Name) + ": " + Encoding.Latin1.GetString(Value);
```

Comment:
Why did this encoding change?

Reply:
This is just for debugging, right? Since we cannot know that's what's intended and this could garble some encodings like UTF-8, I think we should revert it.

Reply:
It is just for debugging. With ASCII you'd only get replacement characters and lose information, at least with Latin1 you'd get representative characters that could be compared to other values. Are you sure you want to revert it?

Reply:
If you think it's useful, I don't feel that strongly.